src/app/components/CardInfo/CardInfo.module.css

```diff
@@ -0,0 +1,55 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
```

Comment:
This is already declared in the globals.css and should not be in here

Reply:
All right. Thank you